prisma/migrations/20240719095833_new_school_structure/migration.sql

```diff
@@ -0,0 +1,28 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `activeCooperation` on the `school` table. All the data in the column will be lost.
+  - You are about to drop the column `emailDomain` on the `school` table. All the data in the column will be lost.
+  - You are about to drop the column `website` on the `school` table. All the data in the column will be lost.
+  - A unique constraint covering the columns `[name,state,city]` on the table `school` will be added. If there are existing duplicate values, this will fail.
+  - Made the column `state` on table `school` required. This step will fail if there are existing NULL values in that column.
```

Comment:
Do we have entries in the schools table?

Reply:
We have around ~20 incomplete entries from 2020/2021.
I'll confirm if we still need any of the info from that table